src/ApplicationInsights.Kubernetes/Debuggings/Inspect.cs

```diff
@@ -0,0 +1,101 @@
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Microsoft.ApplicationInsights.Kubernetes.Debugging
+{
+    /// <summary>
+    /// Diagnostic Source of Application Insights for Kubernetes.
+    /// </summary>
+    public sealed class Inspect
```

Comment:
Yeah, when I think about the name for this class, I am trying to put it together with the namespace. An ideal name would be _Microsoft.ApplicationInsights.Kubernetes.Debugging_.**DiagnosticSource**. Obviously, DiagnosticSource is not a good name since it conflicts.

Despite the length, **ApplicationInsightsKubernetesDiagnosticSource** was another candidate, but then the full name would be: _Microsoft.ApplicationInsights.Kubernetes.Debugging_.**ApplicationInsightsKubernetesDiagnosticSource**.

How about we simply call it **Logger**? And the listener then will be **LogListener**?

Reply:
I think `ApplicationInsightsKubernetesDiagnosticSource`, even if very long, is better than `Logger`. `Logger` suggests that this type is an implementation of `Microsoft.Extensions.Logging.ILogger`, which it is not.

Reply:
Plus, it _is_ a `DiagnosticSource`, so why not call it just that

Reply:
Alright, taken. 👍